man/systemd-sleep.conf.xml

```diff
@@ -78,9 +78,11 @@
         <term>suspend-then-hibernate</term>
 
         <listitem><para>A low power state where the system is initially suspended
-        (the state is stored in RAM). If not interrupted within the delay specified by
-        <command>HibernateDelaySec=</command>, the system will be woken using an RTC
-        alarm and hibernated (the state is then stored on disk).
+        (the state is stored in RAM). In the delay specified by
```

Comment:
maybe let's phrase it as `The system is first suspended for the duration specified by <command>HibernateDelaySec=</command>. After wakeup via an RTC alarm, if the current capacity is Normal, High or Full, the system goes back to suspend, for the amount of time it would take to fully discharge the battery minus 30 minutes, calculated by measuring the discharge rate from before the first suspension to the first wakeup. If the current capacity is Low or Critical, the system will be hibernated (the state is then stored on disk).`